File: examples/java/kora-java-crud-submodule/kora-java-crud-submodule-vet-api/src/main/java/ru/tinkoff/kora/example/submodule/vet/service/VetCache.java
package ru.tinkoff.kora.example.submodule.vet.service;

import ru.tinkoff.kora.cache.annotation.Cache;
import ru.tinkoff.kora.cache.caffeine.CaffeineCache;
import ru.tinkoff.kora.example.submodule.vet.model.dao.Vet;

@Cache("vet-cache")
public interface VetCache extends CaffeineCache<Long, Vet> {

}

File: examples/kotlin/kora-kotlin-crud-submodule/kora-kotlin-crud-submodule-vet-api/src/main/kotlin/ru/tinkoff/kora/kotlin/example/submodule/vet/service/VetCache.kt
package ru.tinkoff.kora.kotlin.example.submodule.vet.service

import ru.tinkoff.kora.cache.annotation.Cache
import ru.tinkoff.kora.cache.caffeine.CaffeineCache
import ru.tinkoff.kora.kotlin.example.submodule.vet.model.dao.Vet

@Cache("vet-cache")
interface VetCache : CaffeineCache<Long, Vet>
